src/data/eurostat/index.csv

```diff
@@ -19,7 +19,7 @@ AL035,,Vlorë
 AT,AT,Austria
 AT1,,Ostösterreich
 AT11,AT_1,Burgenland
-AT111,,Mittelburgenland
+AT111,AT_108,Mittelburgenland
```

Comment:
The keys in this file need updating too

Reply:
Updated, also updated the config.yaml so the Eurostat source runs for location-key AT_*_*